physionet-django/user/management/commands/utility.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+from django.utils import timezone
+
+from user.models import CredentialApplication
+
+
+def get_pending__reference_response_credentialing_application(n, days, ignore_reminded_application=False):
```

Comment:
why the double underscore after pending? this is a loooooooong name! perhaps the fact that it is a management command makes length less of an issue, but something shorter might be nice? I don't have a good alternative unfortunately. Maybe `get_credential_pending_reference` or `get_credential_awaiting_reference`?

Reply:
Ah yes, weird that i put double underscore.

I really like your recommendation. The reason it turned out super long is because, i kind of wanted to make it very clear and ended up overdoing it